api/files.js

```diff
@@ -50,3 +54,88 @@ module.exports.writeMail = function (recipients, template, substitutions) {
 	mkdirp.sync(MAIL_DIRECTORY);
 	return fs.writeFileAsync(filePath, fileContent);
 };
+
+/**
+ * Writes a file to the temp directory. When called in non-development environments,
+ * this method does nothing
+ * @param {Buffer} content		the content to write
+ * @param {Object} params		a parameter object with
+ *                         		{String} key	the key under which to store the file
+ *                         		{String} bucket	the bucket in which to store the file
+ *                         		{String} type	the MIME type of the file
+ * @return {Promise<>}			a resolved promise
```

Comment:
Nope. I meant this as "the result will be undocumented/undefined in its type; do not try to access it." That's because I see no reason why anyone would need to access the `fs` result.